packages/core/integration-tests/test/markdown.js

```diff
@@ -1,24 +1,23 @@
 const assert = require('assert');
 const path = require('path');
-const {bundle, assertBundleTree, outputFS} = require('@parcel/test-utils');
+const {bundle, assertBundles, outputFS} = require('@parcel/test-utils');
 
-describe.skip('markdown', function() {
-  it('should support bundling Markdown', async function() {
+describe('markdown', function() {
+  it.only('should support bundling Markdown', async function() {
```

Comment:
One thing I have also found helpful is using the `--grep` feature in mocha to regex to the specific test I want to run.